docs/architectural-decision-records/017-feature-flag-lifecycles.md

```diff
@@ -22,14 +22,17 @@ A feature typically can be added behind a flag if:
 
 Flags are first added to Launch Darkly via their UI. As an authenticated Launch Darkly user, navigate to our feature flags and click “create flag”. Choose a human readable flag name and a ‘-’ delimited flag key. The flag key is what we use inside our code. Make sure to mark the flag as client side if it is meant to be user visible (rather than API only). If the flag is permanent, mark this as well.
 
-We keep a list of flags in a constants file in `common-code/flags` and all new flags must be added to this file. For example, if you added a flag key of ‘new-testing-flag’, you’d add it to the file as follows:
+We keep a list of flags in a constants file in `app-web/src/common-code/featureFlags` and all new flags must be added to this file. For example, if you added a flag key of ‘new-testing-flag’ with a default value, you’d add it to the file as follows:
```

Comment:
👋 Hey! i don't think this is well suited in this  ADR. An architectural decision record is supposed to be a straightforward comparison of different options.  [More about how Truss uses an ADR.](https://github.com/trussworks/Engineering-Playbook/blob/f0cdfd5e91e6ef70fd6a311ff882ad7281a1aeba/docs/documentation/adr.md)

The only time an ADR is updated (usually) is if we make a different decision.

These docs you are adding are implementation details for testing with LD. This goes in the technical design docs section. That section is for more general docs about how we are going to implement something with a less rigid format.  You could either add to the existing doc on testing approach or if this feels odd there --- feel free to start a launch darkly technical design doc. 

🗒️  I will also take an action item to **add the template we use for ADRs to the repository.**   ADR 0018 was a bit odd because it didn't follow the template but you will see other ADRs follow the template properly and we should just make this explicit so its clearer in the future